example/nextjs/pages/index.js

```diff
@@ -39,7 +39,7 @@ const Index = ({profile, origin}) => (
       <h2 className='subtitle is-4'>
         Stateless authentication microservice for
         <ul>
-          {['Twitter', 'Facebook', 'Google', 'GitHub', 'Reddit', 'LinkedIn', 'Instagram'].map(name => <Item name={name} key={name} />)}
+          {['Twitter', 'Facebook', 'Google', 'GitHub', 'Reddit', 'LinkedIn', 'Instagram', 'Eventbrite'].map(name => <Item name={name} key={name} />)}
```

Comment:
I am in two minds about this
1) If there is a particular strategy thats implemented within the login-with mircoservice, it should follow that there should be a button that will enable that authentication.
2) It is unreasonable to expect everyone to enable buttons for ALL auth strategies 

=> The auth buttons that are needed to be enabled or disabled should be configurable.
Also the URL in the example is hardcoded for it to make a call to login.login-with.com for auth calls, this too should be configurable because it will help others to test their added strategy on their own domain before pushing it unto upstream repo